src/main/java/org/opentripplanner/transit/model/site/StopLocation.java

```diff
@@ -137,4 +142,15 @@ default FeedScopedId getStationOrStopId() {
     }
     return getId();
   }
+
+  static int numberOfStopLocations() {
+    return COUNTER.get();
+  }
```

Comment:
I would call this method `indexCounter()` and and the COUNTER for INDEX_COUNTER - just to prevent this someone from using this to "count number of stop locations". We may get holes in the index in the future...